.travis.yml

```diff
@@ -30,12 +28,12 @@ matrix:
     - php: 5.6
       env: DB=MYSQL BEHAT_TEST=1
     - php: 5.6
+      env: NPM_TEST=1
+  allow_failures:
```

Comment:
Now that we have more than 1 allowed failure test can we use travis' fast finish feature (https://docs.travis-ci.com/user/customizing-the-build/#Fast-Finishing)